extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js

```diff
@@ -220,6 +220,20 @@ describes.realWin('amp-ad-network-doubleclick-impl', realWinConfig, env => {
       expect(fireDelayedImpressionsSpy.withArgs(
           'https://c.com?e=f,https://d.com?g=h', true)).to.be.calledOnce;
     });
+
+    it('should initialize refresh manager', () => {
+      const refreshSpy = sandbox.spy(impl,
+          'initializeRefreshManagerIfEligible_');
+      impl.extractSize({
+        get(name) {
+          return name == 'amp-force-refresh' ? '30' : undefined;
+        },
+        has(name) {
+          return !!this.get(name);
+        },
+      });
+      expect(refreshSpy.withArgs('30')).to.not.be.called;
```

Comment:
You expect it NOT to be called?

Reply:
Fixed.

Reply:
(The reason it passed with the NOT is because it expects a numeric value, rather than a string.)